### PR TITLE
refactor(core): replace the loading state written in useState with useDebouncedValue

### PR DESCRIPTION
**Pull request by JimmFly**
close TOV-856

refactor(core): replace the loading state written in useState with useDebouncedState

fix(core): cloudSvg obscures toggle button

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#6925** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/6925?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/components/pure/cmdk/main.tsx [hunk 1]
```diff
@@ -171,7 +172,7 @@ export const CMDKContainer = ({
   const isInEditor = pageMeta !== undefined;
   const [opening, setOpening] = useState(open);
   const { syncing, progress } = useDocEngineStatus();
-  const [showLoading, setShowLoading] = useState(false);
+  const [showLoading, setShowLoading] = useDebouncedState(false, 500);
```
Comment (pengx17) [inline packages/frontend/core/src/components/pure/cmdk/main.tsx]: oops, maybe you need to use `useDebouncedValue` in this case

Comment (JimmFly) [reply packages/frontend/core/src/components/pure/cmdk/main.tsx]: Do you mean this?
```
 const [showLoading, setShowLoading] = useDebouncedState(false, 500);
 const deferredShowLoading = useDeferredValue(showLoading);
 
 useEffect(() => {
    setShowLoading(syncing);
  }, [syncing, setShowLoading]);
  
  return (
   {deferredShowLoading ? (
   ...
```

Comment (pengx17) [reply packages/frontend/core/src/components/pure/cmdk/main.tsx]: I mean `const deferredShowLoading = useDeferredValue(syncing, 500);`. No need to use `useEffect`.

**Review by Brooooooklyn — APPROVED** [review]

**Pull request merged**